crypto/ec/curve448/curve448utils.h

```diff
@@ -21,10 +21,13 @@
  * alignment on a given platform to support dynamic linking, since even if you
  * header was built with eg arch_neon, you might end up linking a library built
  * with arch_arm32.
+ * Added __SIZEOF_LONG__ check to detect linux_x86_icc.
```

Comment:
This comment isn't needed at this point.

Reply:
Ok.. updated.